cms/djangoapps/contentstore/services.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
Do we want this in the `cms/djangoapps` folder? I expect the automatically generated "suspicious" emails are generated on the LMS side; maybe we should put this in a shared folder?

Reply:
Hmm that's a good point, let me see if there's a better spot for it